bears/python/VultureBear.py

```diff
@@ -19,6 +15,26 @@ class VultureBear:
     ASCIINEMA_URL = 'https://asciinema.org/a/82256'
     CAN_DETECT = {'Unused Code'}
 
-    @staticmethod
-    def create_arguments(filename, file, config_file):
-        return filename,
+    executable = "vulture"
+    arguments = (executable,)
```

Comment:
When you use self.arguments after this you're modifying a class variable. So, that will persist for all objects of the class created after that.